doc/configuration/Configuration.md

```diff
@@ -0,0 +1,319 @@
+## Mattermost Config Settings
+___
+Config settings allow system administrators to customize their teams Mattermost experience. Warning: system administrators should be cautious to only grant admin privileges to team members who understand the critical nature of the settings they are adjusting.
+
+
+## System Console Settings:
+These settings can be modified from the System Console user interface or config.json.
+
+### Service Settings
+___
+General settings to configure the listening address, login security, testing, webhooks and service integration of Mattermost. 
+
+### System
+
+```"ListenAddress": ":8065"```
+*The IP address to listen and the port which to bind. Entering ":8065" will bind to all interfaces or you can choose one like "127.0.0.1:8065". Changing this will require a server restart before taking effect.*
+
+```"MaximumLoginAttempts": 10```
+*Failed login attempts allowed before user is locked out and required to reset password via email.*
```

Comment:
I think proper grammar would be "Failed login attempts allowed before a user is locked out and required to reset their password via email."